recipes/pymcubes/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pymcubes" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyMCubes-{{ version }}.tar.gz
+  sha256: ad25e96f5a32fb5a363372005a4b68eaf03025c1e781ada62e73b10b45ff7cbb
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - python
+    - setuptools
+    - wheel
+    - cython
+    - numpy
```

Comment:
You probably need to repeat cython, numpy, and python on host.

Reply:
Did you mean on `run`? They're already specified as `host` dependencies.

Reply:
Sorry I meant on build.